src/feature/auth/schema/auth.schema.ts

```diff
@@ -1,32 +1,38 @@
-// src/feature/auth/schema/auth.schema.ts
-
 import { z } from 'zod';
 
+import type {
+  AuthResponse,
+  RefreshTokenRequest,
+  RefreshTokenResponse,
+  SignInRequest,
+  SignUpRequest,
+  User,
+} from '@/feature/auth/types/auth.types';
+
+// --- 재사용 가능한 기본 스키마 (클라이언트 및 서버 유효성 검사에 공통 사용) ---
+
 /**
  * 이메일 유효성 검사 스키마
  */
-export const emailSchema = z
-  .string()
-  .min(1, '이메일은 필수 입력입니다.')
-  .email('이메일 형식으로 작성해주세요.');
+
+export const emailSchema = z.string().email('이메일 형식으로 작성해주세요.');
```

Comment:
이 부분 필수 입력 부분이 빠졌습니다 !
```suggestion
export const emailSchema = z
  .string()
  .min(1, '이메일은 필수 입력입니다.')
  .email('이메일 형식으로 작성해주세요.');
```